### PR TITLE
Adding Footer component and updating registeredComponents file

### PR DESCRIPTION
**Pull request by arroya2**
I added a new folder called Footer with three files in it. The first file is the Footer.jsx that will take some arguments and create the footer component. The second file is the FooterOptions.jsx, that is where you can update link locations on the PageOptionsTab. Lastly, the third file is just an image used for previewing the component. And also updated the array to add my footer component into the registeredComponents.js file

### Footer Preview ScreenShots
image

### CODE REVIEW

**Review by WuliuQ — COMMENTED** [review]
I think you need to import your files in registeredComponents.js first, and also make your Footer component folder in the components folder, not just add the Footer after the components folder.

**Comment by Lettuce05** [comment]
[Open in Codeflow](https://pr.new/github.com/cs472-team6/cs472-team6/pull/64)

**Review by CrystaS15 — COMMENTED** [review]
Your component, options, and preview should be in their own Footer folder in the components folder. Your registeredComponents.js should also have imported files as well. When I ran the code, I just saw a blank page with no working features.

File: src/utils/registeredComponents.js
Comment (ZacheryEC001) [inline src/utils/registeredComponents.js:1]: In order for the code to work, you must import Footer, FooterOptions, and FooterPreview as you mention them in the code but never define them, giving a ReferenceError: 'Footer' is not defined.

**Review by ZacheryEC001 — CHANGES_REQUESTED** [review]
Please reference my comment on registeredComponents.js to get your code to work.

**Pull request closed without merging**